spec/features/car_lots/vehicles/index_spec.rb

```diff
@@ -30,25 +30,18 @@
     click_link "create new vehicle"
     expect(page.current_path).to eq("/car_lots/#{fly_guys.id}/vehicles/new")
   end
-end
-# User Story 13, Parent Child Creation (x2)
-
-# As a visitor
-# When I visit a Parent Childs Index page
-# Then I see a link to add a new adoptable child for that parent "Create Child"
-# When I click the link
-# I am taken to '/parents/:parent_id/child_table_name/new' where I see a form to add a new adoptable child
-# When I fill in the form with the child's attributes:
-# And I click the button "Create Child"
-# Then a `POST` request is sent to '/parents/:parent_id/child_table_name',
-# a new child object/row is created for that parent,
-# and I am redirected to the Parent Childs Index page where I can see the new child listed
 
+  it 'should alphabetize the order of vehicles' do 
+    fly_guys = CarLot.create!(name: "Fly Rydes", being_cleaned: true, lot_area: 1200)
+    car1 = Vehicle.create!(name: 'Toyota Yaris', sold: true, price: 7500, car_lot_id: fly_guys.id)
+    car2 = Vehicle.create!(name: 'Chevrolet Silverado', sold: false, price: 10000, car_lot_id: fly_guys.id)
+    visit "/car_lots/#{fly_guys.id}/vehicles"
+    expect(page).to have_link("alphabetical order")
+    click_link("alphabetical order")
+    expect(current_path).to eq("/car_lots/#{fly_guys.id}/vehicles")
+    expect(Vehicle.alphabetize).to eq([car2, car1])
```

Comment:
I like the TDD! But line 42 isn't testing what appears on the page, it is just testing the model method that you defined prior. We need to test how it appears on the page.

With the date test that your mentor helped us write we had a nasty jerry rigged test, but I figured out some of the orderly syntax that helped me clean my test up a bunch. I'll show you the syntax when we meet up later.